src/toPropertyPath.ts

```diff
@@ -0,0 +1,7 @@
+import { decodePointerFragment } from './decodePointerFragment';
+
+export function toPropertyPath(path: string) {
+  return decodePointerFragment(path)
```

Comment:
Do we want to keep regular expression or have a different (more readable?) solution, such as
```
export function toPropertyPath(path: string) {
  return path
    .split('/')
    .filter(Boolean)
    .map(decodePointerFragment)
    .join('.');
}
```